50SAT/SPEX$50SAT.tex

```diff
@@ -0,0 +1,353 @@
+
+The primary deliverable will be the finished, functioning nanosatellite unit.
+In addition, the team should include a report consisting of the following sections:
+\begin{itemize}
+  \item A summary of the development process, including any unforseen issues that arose during development and detailed explanations on how they were addressed.
+  \item Details on any changes to the original design and well as the reasoning for those changes
+  \item Detailed operations guide for handling and operating the satellite
+  \item Where possible, instruction guides for assembly of key components and subsystems that future teams could leverage.
+  \item Test procedures and results for key areas including radio performance, thermal and structural tolerance, and software qualifications
+\end{itemize}
+The published designs also include an arduino-based radio receiver station. The team could develop this component as well, or preferably, leverage other teams and their projects to satisfy this requirement.
+Finally, the team will need to generate a presentation poster that could be used at ImagineRIT or other public SPEX events to highlight the project.
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+If this project is conducted over a semester timeline, the team will need to expediently cover the required areas defined below.
+
+This project will take place over three technical domains: electrical, mechanical, and software.
+
+For the electrical systems, the team will need to review the existing PCB designs, primarily to determine whether it is feasible to manufacture and populate them on campus.
+Also the team will need to determine if all components required are still available, and if not, find suitable replacements and make the necessary design changes.
+PCB assembly will be the system with the longest lead time and should be prioritized first to ensure no bottlenecks on the critical path.
+Once assembled, each will need to be tested, and integrated within the system structure.
+
+For the mechanical systems, the team should evaluate the feasibility of manufacturing the frame on-campus.
+Any tweaks to the internal structure and placement of internal components should be locked down as early as possible.
+The mechanical team will also be responsible for testing the frame and whole assembly for verification against thermal and vibration requirements.
+
+For the onboard software, the team should review the existing code, and try to demonstrate functionality on similar hardware to prevent dependency on the electrical team.
+Once the primary electronics are ready, the software should be tested for functionality on the system hardware.
+
+The timeline below is preliminary:
+
+\begin{table}
+\centering
+ \begin{tabular}{|c | c | c|}
+ \hline
```

Comment:
Besides the style guide "violations" (it's not that bad but i can't think of the right word), the table exceeds the column limits.

Consider making the table extend across both columns at the top of the page.

As an example from my CDIM paper:
```tex
\begin{table*}
  \caption{Critical design requirements for the CDIM spacecraft.}
  \small\centering
  \begin{tabular}{@{}llll@{}} \toprule
    Spacecraft Design Driver & Impact & Target \\ \midrule
    Cost & Science capability, instrument architecture & less than \$\SI{1}{B} \\
    Mass & Launch vehicle & less than \SI{1000}{\kilo\gram} \\
    Temperature (OTA) & Passive radiator & \SI{45}{\kelvin} \\
    Temperature (Detector) & Cryocooler & \SI{35}{\kelvin} \\
    Pointing Requirements & Attitude control system & less than \SI{0.5}{arcsec} \\
    Lifetime & Redundancy, RCS propellant & \SI{5}{years} \\
    Orbit & Solar array, thermal management, launch vehicle, telemetry & Sun-Earth \Ltwo{} \\
    \bottomrule
  \end{tabular}
\label{tab:critical-params}
\end{table*}
```